rainbow-me/rainbow
# Fix hiding create/edit wallet modal

Fixes RNBW-4247
Figma link (if any):

## What changed (plus any additional context for devs)

We use the bottom sheet for this specific modal. It tracks the keyboard position under the hood. But there is a bug. If the keyboard is still visible when we hide the bottom sheet - it would just be stuck. The reason for this is that it checks for the keyboard height and tries to hide the thing behind the keyboard instead. I didn't find any "good props" as a workaround, so now we wait for the keyboard to hide and then close the modal.


## Screen recordings / screenshots


https://user-images.githubusercontent.com/7809008/185952395-fcebcf59-f6a7-4882-b1af-3f29cb530828.mov



## What to test


- Create a wallet then edit it
- Create a new wallet
- The modal should not be stuck

## Final checklist
